packages/core/admin/admin/src/content-manager/components/BlocksEditor/Toolbar/index.js

```diff
@@ -127,7 +127,18 @@ const toggleBlock = (editor, value) => {
     level: level || null,
   };
 
-  Transforms.setNodes(editor, newProperties);
+  const isAboveParagraph =
+    editor.selection && Editor.above(editor, { match: (n) => n.type === 'paragraph' });
```

Comment:
just want to know does it work when you try to check current selection instead of above? because above and current would be the same here as far as I understand.
Also @remidej  is working on breaking out of current node selection for new lines, I dont think above code will work then.

Reply:
I need to check it after the branch of remi will be merged